# HG changeset patch
# User Jaroslav Hajek <highegg@gmail.com>
# Date 1251267428 -7200
#      Wed Aug 26 08:17:08 2009 +0200
# Branch release-3-2-x
# Node ID e7c3e9ad22868c95fe31f42627635b12ed9c7e7b
# Parent  fe30458b1de801fe2a4aac2a02ecd458c01c981a
fix order of args in fmod

diff --git a/src/ChangeLog b/src/ChangeLog
--- a/src/ChangeLog
+++ b/src/ChangeLog
@@ -1,8 +1,12 @@
+2009-08-26  Jaroslav Hajek  <highegg@gmail.com>
+
+	* data.cc (Ffmod): Reverse order of args.
+
 2009-08-25  John W. Eaton  <jwe@octave.org>
 
 	* graphics.cc (gnuplot_backend::send_quit): Wait for gnuplot process.
 
 2009-08-12  Jaroslav Hajek  <highegg@gmail.com>
 
 	* data.cc (Fissorted, F__sort_rows_idx__, Fnorm): Mark as Built-in
 	Functions in the inline help.
diff --git a/src/data.cc b/src/data.cc
--- a/src/data.cc
+++ b/src/data.cc
@@ -1100,18 +1100,18 @@ sign as @var{x}.  If @var{y} is zero, th
 @end deftypefn")
 {
   octave_value retval;
 
   int nargin = args.length ();
 
   if (nargin == 2 && args(0).is_defined () && args(1).is_defined ())
     {
-      octave_value arg_y = args(0);
-      octave_value arg_x = args(1);
+      octave_value arg_x = args(0);
+      octave_value arg_y = args(1);
 
       dim_vector y_dims = arg_y.dims ();
       dim_vector x_dims = arg_x.dims ();
 
       bool y_is_scalar = y_dims.all_ones ();
       bool x_is_scalar = x_dims.all_ones ();
 
       bool is_float = arg_y.is_single_type () || arg_x.is_single_type ();
